taichi/backends/opengl/codegen_opengl.cpp

```diff
@@ -58,6 +58,49 @@ std::string opengl_atomic_op_type_cap_name(AtomicOpType type) {
   return type_names[type];
 }
 
+#if !defined(TI_PLATFORM_WINDOWS)
+#include<sys/wait.h>
+#endif
+
+std::string preprocess_kernel(std::string src) {
+  if (!is_gles()) return src;
+
+#if !defined(TI_PLATFORM_WINDOWS)
```

Comment:
Huh, let's probably undo https://github.com/taichi-dev/taichi/pull/2486 to avoid using macros or `glslc` @ailzhang  ?